public/serverScripts.js

```diff
@@ -58,3 +58,17 @@ async function calc() {
   document.getElementById('answer').innerHTML = `= ${json.answer}`;
   document.getElementById('resultInput').value = '';
 }
+
+document.addEventListener('DOMContentLoaded', function(){
+    var input = document.getElementById('resultInput');
```

Comment:
from linter, extra spacing in the front, 
and we shouldn't use var, in this case since there is no reassignment should use const